doc/visualization.rst

```diff
@@ -2,11 +2,42 @@
 Vector plots
 ------------
 
-We moved from the old "jvec.txt" output file to VTK file 
-format. The information needed for vector plots is in the
-file "jve.vti" and can be visualized using 
+The information needed for vector plots is in the
+file "jvec.{vti,vtu}" and can be visualized using 
 ParaView (https://www.paraview.org/). 
 
+For the visualization of the current vector field we use two different file
+formats.  'vti'/ImageData for grids that consist of orthogonal basis vectors in
+two or three dimensions and evenly spaced grid points, and
+'vtu'/UnstructuredGrid for arbitrary sets of points.  vti/vtu files consist of
+nodes (grid points) and cells.  In vti files the node positions are defined by
+a range and spacing in each dimension and the (cuboid) cells are defined
+implicitly.  In vtu files, the grid point coordinates are given explicitly and
+each cell is defined by a cell type (10 for tetrahedral) and the indices of the
+defining nodes.   For each node we write down a vector (the induced current)
+while the cells are not assigned a value for typical visualizations (ie, the
+value can be set to 0.0).  However, cells need to be defined---paraview
+interpolates the values given at grid points within each cell.
+
+In case of vti files, all required information is given by the grid and no
+further input is required.
+
+In case of vtu files we require an external program to create a tetrahedral
+mesh.  One choice for such a program is TetGen
+(http://wias-berlin.de/software/index.jsp?id=TetGen&lang=1).  The workflow is:
+
+1. obtain a grid of your choice.  The file format should be: no header entries,
```

Comment:
@lnw How do I choose the grid of my choice?

Reply:
I hate to repeat myself, but that's your choice :)

You could use numgrid to obtain something spherical around each atom (but that's a poor/expensive choice for visualisation).  I think the typical usecase is to investigate an oddly shaped molecule and write a custom grid for that.  We could for example have done that for the tori where just the surface +/- a bit is required.

In general, any set of points will do, You could even randomly sample points in whatever volume.

Reply:
It just came to my mind if we have the surface around a molecule we could also do a similar visualization like the ACID community does, namely plotting the current density vector on top of the ACID surface chosen at a specific isovalue.

Reply:
That doesn't sound very thaight forward ... we don't have an ACID isofurface, right?  Can paraview extract isosurfaces from whatever grid (seems plausible)?  If so, one could assign ACID to cells and the rest is probably easy.

Reply:
For an even grid we have the ACID values. A file called acid.vti is created and with Paraview one
can visualize the isosurface. If we have a grid only covering the surface of a molecule it should be possible to visualize both the acid surface and the current density vectors. Otherwise - yes, it is a side discussion and not further relevant for this pull request. We can decide if we want to fill an issue.